### ISSUE TITLE
Node.js backend chapter 7

### ISSUE THREAD

**Issue by maury91** [howtographql/howtographql#758]
After implementing subscriptions, I start on one playground the query

```graphql
subscription {
  newLink {
    node {
      id
      url
      description
      postedBy {
        id
        name
        email
      }
    }
  }
}
```

after launching the query, in the response area, I see a spinner as expected,

on the other playground I run the query

```graphql
mutation {
  post (url: "aaa" description: "bbb") {
    id
    url
    createdAt
    description
  }
}
```

Immediately after running the query, the first playground (the one subscribed) shows the result

```json
{
  "error": {
    "name": "TypeError",
    "message": "Cannot read property 'link' of undefined"
  }
}
```

the playground where I posted the mutation, has shown the correct result (the post with all the expected data)

here are some logs from the node process:

```
Request to https://eu1.prisma.sh/***/hackernews-demo/dev:
query:
subscription ($_v0_where: LinkSubscriptionWhereInput) {
  link(where: $_v0_where) {
    node {
      description
    }
  }
}
operationName: null
variables:
{
  "_v0_where": {
    "mutation_in": [
      "CREATED"
    ]
  }
}
Request to https://eu1.prisma.sh/***/hackernews-demo/dev:
query:
mutation ($_v0_data: LinkCreateInput!) {
  createLink(data: $_v0_data) {
    id
    url
    createdAt
    description
  }
}
operationName: null
variables:
{
  "_v0_data": {
    "description": "bbb",
    "url": "aaa",
    "postedBy": {
      "connect": {
        "id": "cjkpzuyy9k0mg0b308othi0oo"
      }
    }
  }
}
Response from https://eu1.prisma.sh/***/hackernews-demo/dev:
{
  "createLink": {
    "id": "cjkr79zd3ozbe0b30lxp575cq",
    "url": "aaa",
    "createdAt": "2018-08-12T18:46:14.345Z",
    "description": "bbb"
  }
}
Response from https://eu1.prisma.sh/***/hackernews-demo/dev:
{
  "link": {
    "node": {
      "description": "bbb"
    }
  }
}
Response from https://eu1.prisma.sh/***/hackernews-demo/dev:
{
  "link": {
    "node": {
      "id": "cjkr79zd3ozbe0b30lxp575cq",
      "url": "aaa",
      "description": "bbb",
      "postedBy": {
        "id": "cjkpzuyy9k0mg0b308othi0oo",
        "name": "maury",
        "email": "maury@x"
      }
    }
  }
}
```

I see no errors on the node process, I believe there's a chance the error is happening in the playground

**Comment by charmcitygavin** [comment]
I have the exact same issue. I've gone back and double-checked all of my code, even tried different browsers in private mode, but it gives me the same error each time.

```
{
  "error": {
    "name": "TypeError",
    "message": "Cannot read property 'link' of undefined"
  }
}
```

I've gone back to find the `link` that it refers to. Could it be `context.db.subscription.link` in:

```
function newLinkSubscribe(parent, args, context, info) {
    return context.db.subscription.link(
        { where: { mutation_in: ['CREATED'] } },
        info
    )
}
```

Is it possible that `context.db.subscription` has no `link` property?

**Comment by maury91** [comment]
I debugged that line, but seems that `context.db.subscription` includes the values `link` and `user`
<img width="336" alt="screen shot 2018-08-14 at 18 41 15" src="https://user-images.githubusercontent.com/4061875/44108238-b1dbb906-9ff1-11e8-8097-e560407409b8.png">

I find this function hard to debug, mostly because I don't have a good system to debug async iterables.
I can see from the debuger, that when this function is executed https://github.com/graphql/graphql-js/blob/master/src/subscription/subscribe.js#L270-L285 that everything goes fine and the promise isn't rejected, it resolves correctly returning here https://github.com/graphql/graphql-js/blob/master/src/subscription/subscribe.js#L279 showing that `context.db.subscription.link()` returns the expected value (an async iterable), the problem is on the next iteration

**Comment by maury91** [comment]
I was able to trace the error, here is the stack:

```
TypeError: Cannot read property 'link' of undefined
    at Object.<anonymous> (/Users/***/development/hackernews-node/node_modules/graphql-binding/dist/Delegate.js:88:105)
    at step (/Users/***/development/hackernews-node/node_modules/graphql-binding/dist/Delegate.js:32:23)
    at Object.next (/Users/***/development/hackernews-node/node_modules/graphql-binding/dist/Delegate.js:13:53)
    at fulfilled (/Users/***/development/hackernews-node/node_modules/graphql-binding/dist/Delegate.js:4:58)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:182:7)
```

I catched that error here: https://github.com/graphql/graphql-js/blob/master/src/subscription/mapAsyncIterator.js#L29

I will debug more

**Comment by maury91** [comment]
Issue is on `prisma-bindings` https://github.com/prisma/prisma-binding/issues/229

**Comment by maury91** [comment]
downgrading to "prisma-binding" "2.1.3" has solved the issue.

```bash
rm -rf node_modules && unlink yarn.lock && yarn add prisma-binding@2.1.3 && yarn
```

**Comment by maury91** [comment]
This is fixed and released in prisma-binding 2.1.5 🎉
